charts/tf-controller/README.md

```diff
@@ -1,6 +1,6 @@
 # Weave GitOps Terraform Controller
 
-![Version: 0.15.2](https://img.shields.io/badge/Version-0.15.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.15.2](https://img.shields.io/badge/AppVersion-v0.15.2-informational?style=flat-square)
+![Version: 0.15.3](https://img.shields.io/badge/Version-0.15.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.15.3](https://img.shields.io/badge/AppVersion-v0.15.3-informational?style=flat-square)
```

Comment:
Please bump version to v0.16.0-rc.1, as we'll ship the branch planner in this version.